deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/layers/normalization/KerasBatchNormalization.java

```diff
@@ -42,6 +42,8 @@
     private final String PARAM_NAME_BETA = "beta";
     private final String PARAM_NAME_RUNNING_MEAN = "running_mean";
     private final String PARAM_NAME_RUNNING_STD = "running_std";
+    private final String PARAM_NAME_MOVING_MEAN = "moving_mean";
```

Comment:
actually, when parameter names differ between keras 1 and 2 we reflect that through `KerasLayerConfiguration`. Please put `PARAM_NAME_MOVING_MEAN` and `PARAM_NAME_MOVING_VARIANCE` there (empty) and overwrite it in `Keras1LayerConfiguration` and `Keras2LayerConfiguration`, respectively. Each `KerasLayer` has a `conf` and you can get these parameters through lombok getters. This way we don't have these if-else statements all over the place.

In your situation it's a little more overhead, but it makes differing configs explicit in one central place.